pyproject.toml

```diff
@@ -49,6 +49,7 @@ extras = [
     "hgvs>=1.4",
     "dill~=0.3.7",
     "click",
+    "requests",
```

Comment:
I don't think this is needed since it's included in required `dependencies` in line 40
```suggestion
```

Reply:
Yup, missed that